packages/common/src/TimeUtils.js

```diff
@@ -14,6 +16,16 @@ const averageBlockTimeSeconds = async (/* lookbackSeconds */) => {
   }
 };
 
+// Sets fromBlock to the value of an environment variable if one is set. This can be set to 0 to make tests work with Ganache, or any other value needed for a production script or bot.
+async function getFromBlock() {
```

Comment:
nit: does this function need to be async?

Reply:
Nope! It was async from earlier when I was looking at getting the block number from the node, before changing tracks.

Reply:
+1 for not making this async